docs/taskruns.md

```diff
@@ -698,11 +699,25 @@ object(s), if present. Any `Request` or `Limit` specified by the user (on `Task`
 
 For more information, see the [`LimitRange` support in Pipeline](./compute-resources.md#limitrange-support).
 
+### Specifying `Retries`
+You can use the `retries` field to set how many times you want to retry on a failed TaskRun.
+All TaskRun failures are retriable except for `Cancellation`.
+
+For a retriable `TaskRun`, when an error occurs:
+- The error status is archived in `.status.RetriesStatus`
+- The `Succeed` condition in `.status` is updated: 
+```
+Type: Succeed
+Status: Unknown
+Reason: CompleteWithRetries
+```
+- `status.StartTime` and `status.PodName` are unset to trigger another retry attempt.
+
 ### Configuring the failure timeout
 
-You can use the `timeout` field to set the `TaskRun's` desired timeout value. If you do not specify this
-value for the `TaskRun`, the global default timeout value applies. If you set the timeout to 0, the `TaskRun` will
-have no timeout and will run until it completes successfully or fails from an error.
+You can use the `timeout` field to set the `TaskRun's` desired timeout value for each retry attempt. If you do
+not specify this value for the `TaskRun`, the global default timeout value applies. If you set the timeout to 0,
```

Comment:
Please clarify how the global default timeout applies: does the global timeout apply _to each retry_ or does the global timeout apply _to the sum of all retries_?

Reply:
SG~ updated to make it clearer!

Just to clarify, the doc here illustrates that the global timeout **value** applies, so the behavior is the same. But it's always good to be explicit.